### DISCUSSION TITLE
Standard file encoding for hospital MRFs

### DISCUSSION THREAD

**Discussion by ravi-pinapaka** [CMSgov/hospital-price-transparency#96] [Q&A] [answered]
Is there any requirement for a standard file encoding for the hospital MRFs? While some of the files which are already compliant with the schema are encoded in UTF-8, many of them have a different encoding such as windows-1252. For instance, the CMS example file for the CSV wide format has characters which cannot be parsed by a UTF-8 decoder.

Can you please let me know how the files are required to be encoded to allow their consumption without any issue or data loss?

**Comment by RidenourK** [comment, marked as answer]
@ravi-pinapaka. Thank you for your question. 

The headers/attributes in hospital MRFs must conform to UTF-8 and the form and manner requirements specified in the Hospital Price Transparency Data Dictionary GitHub repository in order to pass the validator.
